### ISSUE TITLE
Crash - Memory access violation 0x77c178c0

### ISSUE THREAD

**Issue by GoogleCodeExporter** [marig345/prpltwtr#27]
```
Client (Pidgin, finch, empathy, etc): Pidgin
Client Version: 2.7.1
LibPurple Version: 2.7.1
PrplTwtr Version: 0.5.2
Operating System: Win XP SP3

Problem/Feature request (Be as specific as possible. Steps to reproduce are
especially helpful):
Nothing, juste wait. Twitter "chat room" always opened

Pidgin crash report included
```

Original issue reported on code.google.com by `mem...@lenglet.name` on 6 Jul 2010 at 9:52

Attachments:
- [pidgin.RPT](https://storage.googleapis.com/google-code-attachments/prpltwtr/issue-27/comment-0/pidgin.RPT)

**Comment by GoogleCodeExporter** [comment]
```
I'm pretty sure this was fixed in 0.6.0. Can you see if you can reproduce 
[also, please install the .dbgsym for this version]
```

Original comment by `mikeage` on 14 Feb 2011 at 6:52
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `mikeage` on 23 Jul 2011 at 6:25
- Changed state: **Invalid**
